iacrcc/iacrcc.cls

```diff
@@ -486,7 +486,7 @@ xscale=\@OrigHeightRecip*\@curXheight,transform shape]
         \else
           \setcounter{footnote}{0}%
           \setcounter{IACR@author@cnt}{1}%
-          \def\and{\if@IACR@autoinst\inst{\theIACR@author@cnt} \fi
+          \def\and{%
```

Comment:
I was immediately confused - what is \if@IACR@autoinst supposed to represent anyway? Can we add a comment since the name is not very descriptive? It's also not clear to me what problem this solves. Is it related to an existing issue?

Reply:
Agreed that this is not very clear. The while Maketitle has *a lot* of legacy code where I suspect this came from. The @IACR@autoinst inserts automatically \inst numbering when needed: this is actually something we do not want. I will create a new issue to clean-up / re-write this with proper comments.
This commit is fixing Issue #122.